README.md

```diff
@@ -1,4 +1,4 @@
-# Earth Enterprise [![Chat on Slack](https://img.shields.io/badge/chat-on%20slack-ff69b4.svg)](http://slack.opengee.org)
+# Earth Enterprise [![Chat on Slack](https://img.shields.io/badge/chat-on%20slack-ff69b4.svg)](http://slack.opengee.org) [![build](https://travis-ci.org/tst-eclamar/earthenterprise.svg?branch=master)](https://travis-ci.org/tst-eclamar/earthenterprise/builds)
```

Comment:
We'll need to change tst-eclamar to google in two places on this line, though we may want to hold off on the badge until we get Google to turn it on.